### PR TITLE
Tests: Add Fedora-26 to packaging tests

### PR DESCRIPTION
**Pull request by dliappis**
Add Fedora-26 to packaging tests. Don't retire Fedora-25 yet, as it
will be EOL, one month after Fedora-27 is out.

### CODE REVIEW

**Comment by dakrone** [comment]
This failed for me with this log: http://termbin.com/wtgf

**Comment by dliappis** [comment]
@dakrone Thanks for testing this out! Perhaps you were testing this directly from my fork+branch which was not 100% up-to-date with elastic/elasticsearch#master. I merged elastic/elasticsearch#master in my fork and the tests pass fine for me:

```
$ gradle --stacktrace clean
$ gradle --info -Pvagrant.boxes=fedora-26 packagingTest

<snip here>

ok 223 [REINSTALL] install
ok 224 [REINSTALL] purge elasticsearch
ok 225 [REINSTALL] chown directories
ok 226 [REINSTALL] reinstall elasticsearch
ok 227 [REINSTALL] check ownership
:qa:vagrant:vagrantFedora26#packagingTest (Thread[main,5,main]) completed. Took 9 mins 38.962 secs.
:qa:vagrant:vagrantFedora26#halt (Thread[main,5,main]) started.
:qa:vagrant:vagrantFedora26#halt
Putting task artifact state for task ':qa:vagrant:vagrantFedora26#halt' into context took 0.0 secs.
Executing task ':qa:vagrant:vagrantFedora26#halt' (up-to-date check took 0.0 secs) due to:
  Task has not declared any outputs.
Starting process 'command 'vagrant''. Working directory: /home/dliappis/dl/elasticsearch/qa/vagrant Command: vagrant halt fedora-26
Successfully started process 'command 'vagrant''
==> fedora-26: Removing cache buckets symlinks...
==> fedora-26: Attempting graceful shutdown of VM...
:qa:vagrant:vagrantFedora26#halt (Thread[main,5,main]) completed. Took 7.186 secs.
:qa:vagrant:packagingTest (Thread[main,5,main]) started.
:qa:vagrant:packagingTest
Skipping task ':qa:vagrant:packagingTest' as it has no actions.
:qa:vagrant:packagingTest (Thread[main,5,main]) completed. Took 0.0 secs.

BUILD SUCCESSFUL

Total time: 14 mins 13.886 secs
Stopped 0 worker daemon(s).
```

**Review by dakrone — APPROVED** [review]
LGTM, this passed once I merged master in, thanks @dliappis!

**Comment by dliappis** [comment]
Backported:

5.6 -> 991183de370363f848b4d1d5062479b6a7cad541
6.0 -> 7e35354028abcdfd6f7be5455a5b6bdb891af3cf
6.x -> 9e32206c58da090651d48aded20866372ab278a2

**Pull request merged**
